.standard.yml

```diff
@@ -0,0 +1,3 @@
+fix: true
+parallel: true
+format: progress
```

Comment:
Lastly, just for my information, are these settings for some tool/CI process? Since I am not aware of these.

Reply:
This is a config for https://github.com/standardrb/standard. With this config it will now automatically fix lint errors when running `bundle exec standardrb` and use all cores to run it in parallel